addons/mail/models/mail_thread.py

```diff
@@ -2725,18 +2725,48 @@ def _notify_get_reply_to(self, default=None, records=None, company=None, doc_nam
                 if catchall:
                     result_email.update(dict((rid, '%s@%s' % (catchall, alias_domain)) for rid in left_ids))
 
-            # compute name of reply-to - TDE tocheck: quotes and stuff like that
-            company_name = company.name if company else self.env.company.name
-            for res_id in result_email.keys():
-                name = '%s%s%s' % (company_name, ' ' if doc_names.get(res_id) else '', doc_names.get(res_id, ''))
-                result[res_id] = tools.formataddr((name, result_email[res_id]))
+            for res_id in result_email:
+                result[res_id] = self._notify_get_reply_to_formatted_email(
+                    result_email[res_id],
+                    doc_names.get(res_id) or '',
+                    company or self.env.company
+                )
 
         left_ids = set(_res_ids) - set(result_email)
         if left_ids:
             result.update(dict((res_id, default) for res_id in left_ids))
 
         return result
 
+    def _notify_get_reply_to_formatted_email(self, record_email, record_name, company):
```

Comment:
we have only this bug for the reply-to ? (not for email-from, email-to ?)

Reply:
As discussed, yes :)